Sprint-1/errors/2.js

```diff
@@ -3,3 +3,6 @@
 
 console.log(`I was born in ${cityOfBirth}`);
 const cityOfBirth = "Bolton";
+
+//ReferenceError: Cannot access 'cityOfBirth' before initialization. in javaScript variable that declare with const and let can not access before assigning theme
```

Comment:
So, this is a very very important topic not only JavaScript, but in any programming language. In short, accessing variables and **where** they can be accessed from is what we call _variable scope_. I will leave this introductory and short [article](https://www.codecademy.com/resources/docs/javascript/variable-scope) with you, so you can get familiarised with what we mean by scope and what different "points" of access are for our variables. Scope will also become really important when you eventually move to writing your own functions and later on, defining your classes! 😊

Topics like variable scope or scope in general are one of the most fundamental aspects of programming and making our code run as efficiently as possible, so it's really important to understand this. 

As an added bonus, although this is a slighly more advanced subject, I would also suggest for you to maybe start playing around understand what [hoisting](https://developer.mozilla.org/en-US/docs/Glossary/Hoisting) is in JavaScript and what is the different between `const`, `let` and now pretty much redundant `var` declaration of variables. If you understand what hoisting is, you will pretty much understand how JavaScript works under the hood.

Reply:
Hi @giorgigutsaevi, thanks for your review! I completely agree with your points. I'm actively working on strengthening my understanding of these topics and will make an effort to deepen my grasp of hoisting. Thanks so much again!